### ISSUE TITLE
MemoryBytePatternSearcher progress stuck at 100%

### ISSUE THREAD

**Issue by astrelsky** [NationalSecurityAgency/ghidra#2074]
**Describe the bug**
The TaskMonitor progress of MemoryBytePatternSearcher stays at 100% throughout the search.

**To Reproduce**
1. Run the "Windows x86 PE RTTI Analyzer" on a large PE with and numerous RTTI0 and RTTI4 models.

**Environment (please complete the following information):**
 - OS: Microsoft Windows [Version 10.0.19041.329]
 - Java Version: 14.0.1
 - Ghidra Version: 9.2 c5a31bb129944560f7d70deccb836c4f63349d82

**Comment by dev747368** [comment]
To be clear, the action completes, but while its doing its stuff, the progress bar is bunk?

**Comment by astrelsky** [comment]
> To be clear, the action completes, but while its doing its stuff, the progress bar is bunk?

Yes it did complete eventually. It took a few hours though. I didn't have the debugger attached at the time but I could tell it wasn't stuck because of my laptop fan.
